consensus/istanbul/core/events.go

```diff
@@ -30,4 +31,6 @@ type backlogEvent struct {
 	Hash common.Hash
 }
 
-type timeoutEvent struct{}
+type timeoutEvent struct {
+	view *istanbul.View
```

Comment:
To make this clearer.
```suggestion
	nextView *istanbul.View
```

Reply:
Applied, thanks.